README.md

```diff
@@ -4790,6 +4790,35 @@ require'lspconfig'.sumneko_lua.setup{
 
 If you install via our installer, if you execute `:LspInstallInfo sumneko_lua`, you can know `cmd` value.
 
+The settings of the language server can also be overridden. This is especially useful
+when developing on the lua components of neovim. Some recommended settings are as follows:
+
+```lua
+require'lspconfig'.sumneko_lua.setup {
+  settings = {
+    Lua = {
+      runtime = {
+        -- Tell the language server which version of Lua you're using (LuaJIT in the case of Neovim)
```

Comment:
It can be used with plain lua too so I would add a `most likely`.

Reply:
addressed